src/backend/api/reviews.js

```diff
@@ -35,15 +35,15 @@ router.get("/", async (request, response) => {
   router.put("/:id", async (request, response) => {
     try {
       const updateReview = await knex("review").where({id:request.params.id}).update(request.body);
-      response.json(updateReview);
+      response.json("this review has been updated");
```

Comment:
now you're sending a string instead of a json object so I'll advice you send ``response.send("this review has been updated")`` and make the corrections in all the places, where you have the same string reply and not an object.